frontend/components/ViewAllHostsLink/ViewAllHostsLink.tsx

```diff
@@ -13,6 +13,8 @@ interface IHostLinkProps {
   platformLabelId?: number;
   /** Shows right chevron without text */
   condensed?: boolean;
+  /** Table links shows on row hover only */
+  rowHover?: boolean;
```

Comment:
Note: This new prop adds "row-hover-link" classname to hide the link and show on row hover only